### PR TITLE
format the code

### PR DESCRIPTION
**Pull request by geowa4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,6 @@ Please open a github issue including cast version number `cast --version`.
 
 Pull requests are gratefully received!
 
-- please 'gofmt' the code.
```
Comment (barnybug) [inline README.md]: Not sure why you're removing this request for PRs to be gofmt'ed?

Comment (geowa4) [reply README.md]: Ohhhh, sorry. I thought the README was requesting someone gofmt the existing code. I’ll put that back.

**Comment by geowa4** [comment]
Nah, it's not a Hacktoberfest PR; I had those done a while ago. 🥇 I reverted the change to the README. Sorry for the misunderstanding on that one.

**Comment by geowa4** [comment]
By the way, I really like this tool. It's helped me a bunch.

**Comment by barnybug** [comment]
Merged. thanks.

**Pull request closed without merging**